queue_job/job.py

```diff
@@ -891,6 +891,26 @@ def related_action(self):
         action_kwargs = self.job_config.related_action_kwargs
         return action(**action_kwargs)
 
+    def error_handler(self, exc):
+        record = self.db_record()
+        funcname = self.job_config.error_handler_func_name
+        if not self.job_config.error_handler_enable or not funcname:
+            return None
+
+        if not isinstance(funcname, str):
+            raise ValueError(
+                "error_handler must be the name of the method on queue.job as string"
+            )
+        action = getattr(record, funcname)
+        _logger.info("Job %s fails due to %s, execute %s", self.uuid, exc, action)
+        action_kwargs = self.job_config.error_handler_kwargs
+        action_kwargs["job"] = self
+        action_kwargs["job"] = self
```

Comment:
Duplicate line.

Can also be written as:
```suggestion
        action_kwargs = {**self.job_config.error_handler_kwargs, "job": self}
```